### PR TITLE
More editorial improvements

### PR DESCRIPTION
**Pull request by ptomato**
- Addressing editorial issues from the backlog
- Consistency in phrasing, both within Temporal and with ECMA-262
- More structured headers

### CODE REVIEW

File: spec/plainyearmonth.html [hunk 1]
```diff
@@ -520,21 +553,32 @@ <h1>
...
 
-    <emu-clause id="sec-temporal-regulateisoyearmonth" aoid="RegulateISOYearMonth">
-      <h1>RegulateISOYearMonth ( _year_, _month_, _overflow_ )</h1>
+    <emu-clause id="sec-temporal-regulateisoyearmonth" type="abstract operation">
+      <h1>
+        RegulateISOYearMonth (
+          _year_: an integer,
+          _month_: an integer,
+          _overflow_: *"constrain"* or *"reject"*,
+        ): either a normal completion containing an ISO Year-Month Record, or a throw completion
```
Comment (Ms2ger) [inline spec/plainyearmonth.html:562]: Does this need a comma?

File: spec/zoneddatetime.html [hunk 2]
```diff
@@ -1553,6 +1554,7 @@ <h1>
         1. Return ! CreateTemporalDuration(_sign_ &times; _result_.[[Years]], _sign_ &times; _result_.[[Months]], _sign_ &times; _result_.[[Weeks]], _sign_ &times; _result_.[[Days]], _sign_ &times; _timeResult_.[[Hours]], _sign_ &times; _timeResult_.[[Minutes]], _sign_ &times; _timeResult_.[[Seconds]], _sign_ &times; _timeResult_.[[Milliseconds]], _sign_ &times; _timeResult_.[[Microseconds]], _sign_ &times; _timeResult_.[[Nanoseconds]]).
       </emu-alg>
     </emu-clause>
+
     <emu-clause id="sec-temporal-adddurationtoOrsubtractdurationfromzoneddatetime" type="abstract operation">
```
Comment (Ms2ger) [inline spec/zoneddatetime.html:1558]: Feel free to lower-case the O in this id as well

**Review by Ms2ger — APPROVED** [review]
Thanks!

File: spec/zoneddatetime.html (hunk 2 above)
Comment (ptomato) [reply spec/zoneddatetime.html]: Added another commit fixing this and two others that I found with `git grep`.

**Comment by ptomato** [comment]
Nice catches, thanks for taking such a thorough look.

**Pull request merged**